packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -141,17 +142,17 @@ class TerraService {
    * Uploads the reference screenshots to the remote repository if this build was triggered from a PR merge.
    * @param {Object} config wdio configuration object
    */
-  // async onComplete(_, config) {
-  //   try {
-  //     if (this.serviceOptions.useRemoteReferenceScreenshots /* && This is from a PR merge  */ ) {
-  //       const screenshotConfig = getRemoteScreenshotConfiguration(config.screenshotsSites, this.serviceOptions.buildBranch);
-  //       const screenshotRequestor = new ScreenshotRequestor(screenshotConfig.publishScreenshotConfiguration);
-  //       await screenshotRequestor.upload();
-  //     }
-  //   } catch (error) {
-  //     throw new SevereServiceError(error);
-  //   }
-  // }
+  async onComplete(_, config) {
+    try {
+      if (this.serviceOptions.useRemoteReferenceScreenshots && this.serviceOptions.buildBranch === BUILD_BRANCH.master && this.serviceOptions.buildType === BUILD_TYPE.branchEventCause) {
```

Comment:
We should not restrict the screenshot upload to the `master` branch. Screenshot upload during the PR merge build should be performed for any branch. For example, if my PR is targeted to be merged into `dev` and any feature branch, I would still expect the screenshots to be uploaded to this targeted branch after the PR is merged.

Reply:
ah okay, then i need to change this from (`is master` && `is BranchEvent`) to (`is not pr-#` && `is BranchEvent`)